source/_data/themes.yml

```diff
@@ -2955,3 +2955,13 @@
     - fast
     - clean
     - simple
+- name: July
+  description: A minimalist, focused writing Hexo theme.
+  link: https://github.com/wisp-x/hexo-theme-july
+  preview: https://github.com/wisp-x/hexo-theme-july
+  tags:
+    - hexo-theme
+    - responsive
+    - minimalism
+    - simple
+    - en
```

Comment:
Please add a new line at the end of the file.

Reply:
done